test/aqua/test_vqc.py

```diff
@@ -390,32 +304,18 @@ def test_vqc_on_wine(self, mode):
                                                 test_size=testing_dataset_size,
                                                 n=feature_dim,
                                                 plot_data=False)
-        ref_accuracy = 0.2
-        if mode == 'wrapped':
-            warnings.filterwarnings('ignore', category=DeprecationWarning)
-            data_preparation = SecondOrderExpansion(feature_dim)
-            wavefunction = RYRZ(feature_dim, depth=1)
-        else:
-            data_preparation = ZZFeatureMap(feature_dim)
-            wavefunction = TwoLocal(feature_dim, ['ry', 'rz'], 'cz', reps=1, insert_barriers=True)
-            if mode == 'circuit':
-                data_preparation = QuantumCircuit(feature_dim).compose(data_preparation)
-                wavefunction = QuantumCircuit(feature_dim).compose(wavefunction)
+        aqua_globals.random_seed = self.seed
+        data_preparation = ZZFeatureMap(feature_dim)
+        wavefunction = TwoLocal(feature_dim, ['ry', 'rz'], 'cz', reps=1, insert_barriers=True)
 
         vqc = VQC(COBYLA(maxiter=100), data_preparation, wavefunction, training_input, test_input)
+        result = vqc.run(self.statevector_simulator)
 
-        if mode == 'wrapped':
-            warnings.filterwarnings('always', category=DeprecationWarning)
-
-        result = vqc.run(QuantumInstance(BasicAer.get_backend('statevector_simulator'),
-                                         shots=1024,
-                                         seed_simulator=aqua_globals.random_seed,
-                                         seed_transpiler=aqua_globals.random_seed))
-        self.assertGreater(result['testing_accuracy'], ref_accuracy)
+        self.log.debug(result['testing_accuracy'])
+        self.assertLess(result['testing_accuracy'], 0.6)
```

Comment:
Since this is accuracy it would seem we want to ensure it was some minimal level to assure things were working - before it was assertGreater which seems ok, assertLess seems like an incorrect choice in this case. If it were checking for some loss apect then I could understand